### PR TITLE
Get topics

### PR DESCRIPTION
**Pull request by akin-fagbohun**
database and testing is up and running. Review for first GET endpoint ready for merge with main branch.

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe("GET /api/topics", () => {
+  test('sends GET to endpoint -> responds with object with two properties', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then((res) => {
+        expect(res.body[0]).toMatchObject({
```
Comment (karlaferry) [inline __tests__/app.test.js]: I would make sure that I test all of the objects inside the array and not just the index zero! You could make use of a forEach + expect.any just to check that the properties exist and the values are in correct data types. No need to check exact values for now.

File: index.js [hunk 2]
```diff
@@ -0,0 +1,29 @@
+const express = require('express');
+const db = require('./db/connection');
+
+const app = express();
+
+// middleware
+app.use(express.json());
+
+app.get('/api/topics', async (req, res, next) => {
```
Comment (karlaferry) [inline index.js]: Please extract your controllers and models!

**Review by karlaferry — CHANGES_REQUESTED** [review]
I've left some comments on improving this so have a look at them please :) Submit it again for review when you're done! Looks good so far though 👍🏼

**Comment by cavalierswain** [comment]
Good work on implementing the feedback Akin, nicely done!

**Pull request merged**